diff --git a/src/main.cpp b/src/main.cpp
index d051197..93c2373 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -618,7 +618,7 @@ int main(int argc, char** argv) {
             maxfd = std::max(maxfd, toxvpn.listener->populate_fdset(&readset));
 #endif
 #endif
-        int interval = tox_iteration_interval(my_tox);
+        int interval = 1;
 #ifdef USE_SELECT
         timeout.tv_sec = 0;
         timeout.tv_usec = interval * 1000;
